<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RightCameraWgt</class>
 <widget class="QWidget" name="RightCameraWgt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="TGFrame" name="rightVideoFrame1" native="true">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>50</y>
     <width>291</width>
     <height>211</height>
    </rect>
   </property>
   <widget class="OpenGLWidget" name="playWgt" native="true">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>21</y>
      <width>249</width>
      <height>169</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="TGFrame" name="rightTitleFrame1" native="true">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>50</y>
     <width>311</width>
     <height>211</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="leftMargin">
     <number>20</number>
    </property>
    <property name="topMargin">
     <number>20</number>
    </property>
    <property name="rightMargin">
     <number>20</number>
    </property>
    <property name="bottomMargin">
     <number>20</number>
    </property>
    <item>
     <widget class="QWidget" name="taskInfoWgt" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="TGFrame" name="rightTitleFrame2" native="true">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>270</y>
     <width>411</width>
     <height>291</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <property name="leftMargin">
     <number>20</number>
    </property>
    <property name="topMargin">
     <number>20</number>
    </property>
    <property name="rightMargin">
     <number>20</number>
    </property>
    <property name="bottomMargin">
     <number>20</number>
    </property>
    <item>
     <widget class="QWidget" name="doInfoWgt" native="true">
      <widget class="QLabel" name="stopOverLab">
       <property name="geometry">
        <rect>
         <x>49</x>
         <y>160</y>
         <width>84</width>
         <height>32</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>停靠完成</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="exitQueueLab">
       <property name="geometry">
        <rect>
         <x>223</x>
         <y>160</y>
         <width>84</width>
         <height>32</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>退出排队</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="exitQueueImg">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>70</y>
         <width>71</width>
         <height>81</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="stopOverImg">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>70</y>
         <width>71</width>
         <height>81</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TGFrame</class>
   <extends>QWidget</extends>
   <header>Component/TGFrame.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>OpenGLWidget</class>
   <extends>QWidget</extends>
   <header>Component/OpenglWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
